packages/vuetify/src/components/VTextField/VTextField.ts

```diff
@@ -371,6 +371,9 @@ export default baseMixins.extend<options>().extend({
         style: {
           width: !this.isSingle ? convertToUnit(width) : undefined,
         },
+        attrs: {
+          align: this.reverse ? 'right' : 'left',
```

Comment:
I would've asked you to redo with CSS anyway, this attribute is deprecated.

Reply:
Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1488228

Looks like using margin instead of text-align works in firefox.

Reply:
Thanks @KaelWD , Will verify the same and update the PR

Reply:
@KaelWD As mentioned in the Mozilla patch here [https://bugzilla.mozilla.org/attachment.cgi?id=9006391&action=edit](url)

![image](https://user-images.githubusercontent.com/26546015/106858520-cd227c80-66e7-11eb-853a-a2e52718e393.png)


These changes seems to fix it.